package.json

```diff
@@ -393,6 +393,13 @@
           "os": "iOS 15.0"
         }
       },
+      "simulator.17.0": {
```

Comment:
Additional e2e test changes will be needed to run iOS 17.0 in CI as the keyboard is displayed on this emulator by default. This will be addressed in a separate PR.

Reply:
Removed in 4820351b8c175323807ad646f5206f22ecf06917!